deps/first/Helpers/src/Win32Error.cpp

```diff
@@ -0,0 +1,78 @@
+#ifdef _WIN32
+#include <windows.h>
+#include <format>
+
+#include <String/StringType.hpp>
+
+#include "Helpers/Win32Error.hpp"
+
+namespace RC
+{
+    Win32Error::Win32Error(DWORD errorCode)
+    {
+        constexpr auto dwLanguageId = MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT);
```

Comment:
```suggestion
        static constexpr auto dwLanguageId = MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT);
```
If your intention was to make this a compile-time constant, use `static constexpr`.
Non-static constexpr doesn't guarantee it's going to be optimized away at compile-time.
Here's an example of non-static constexpr:
![image](https://github.com/user-attachments/assets/a003c7de-e1b0-4048-b197-917e7dd78183)
And here's static constexpr with the same compile options:
![image](https://github.com/user-attachments/assets/7c4e7d3b-6473-41ff-970a-d21a7db3b884)